src/components/dev-dot-content/mdx-components/mdx-tabs/mdx-tabs.module.css

```diff
@@ -0,0 +1,3 @@
+.tabsWrapper {
+  margin: 24px 0;
```

Comment:
This was added in #387 , but only for tutorials. Now applies to both tutorials and docs.